src/views/dialog.rs

```diff
@@ -14,8 +14,13 @@ use view::{AnyView, Selector, View};
 use views::{Button, DummyView, SizedView, TextView};
 
 #[derive(PartialEq)]
-enum Focus {
+/// Identifies currently focused elements in [`Dialog`].
+///
+/// [`Dialog`]: struct.Dialog.html
+pub enum Focus {
```

Comment:
I think we should rename this to `DialogFocus`, otherwise, it's not clear from `cursive::views::Focus` what this refers to.

Reply:
It was one of mine concerns.. . I'll change it.